engine/src/main/java/org/terasology/engine/persistence/serializers/NetworkEntitySerializer.java

```diff
@@ -218,7 +218,11 @@ public void deserializeOnto(MutableComponentContainer entity, EntityData.PackedE
                 byte fieldId = entityData.getFieldIds().byteAt(fieldPos);
                 ReplicatedFieldMetadata fieldMetadata = metadata.getField(fieldId);
                 if (fieldMetadata != null && fieldCheck.shouldDeserialize(metadata, fieldMetadata)) {
-                    logger.trace("Deserializing field {} of component {} as value {}", fieldMetadata, metadata, entityData.getFieldValue(fieldPos));
+                    logger.atTrace().
+                            addArgument(() -> fieldMetadata).
+                            addArgument(() -> metadata).
+                            addArgument(entityData.getFieldValue(fieldPos)).
+                            log("Deserializing field {} of component {} as value {}");
```

Comment:
`entityData.getFieldValue(fieldPos)` will be called in next line anyway so create and reference local variable